content/courses/onchain-development/intro-to-anchor.md

```diff
@@ -156,13 +183,18 @@ mod program_module_name {
 }
 
 #[derive(Accounts)]
-pub struct InstructionAccounts {
-    #[account(init, payer = user, space = 8 + 8)]
+pub struct InstructionAccounts<'info> {
+    #[account(
+        init,
+        payer = user,
+        space = (DISCRIMINATOR as usize) + AccountStruct::INIT_SPACE
```

Comment:
Just make DISCRIMINATOR a usize in the first place! Also call it ANCHOR_DISCRIMINATOR since that's what we've used elsewhere.